### DISCUSSION TITLE
can i skid fdp

### DISCUSSION THREAD

**Discussion by ThanhVietnam2011** [SkidderMC/FDPClient#926] [Q&A]
can i skid fdp

**Comment by DavidDavid1337** [comment]
Yes its open source just paste it

**Reply by ThanhVietnam2011** [reply]
so how can i skid it

**Reply by DavidDavid1337** [reply]
just paste the code

**Reply by DavidDavid1337** [reply]
ctrl + c ctrl + v

**Reply by ThanhVietnam2011** [reply]
i mean like this video https://www.youtube.com/watch?v=mpp2B0gZXFE&t=81s but for version 1.8.9
it uses gradle and it for 1.12.2 not for 1.8.9 
so i would like to ask you if it is possible to skid fdp with gradle

**Reply by ManInMyVan** [reply]
> i mean like this video https://www.youtube.com/watch?v=mpp2B0gZXFE&t=81s but for version 1.8.9
> it uses gradle and it for 1.12.2 not for 1.8.9 
> so i would like to ask you if it is possible to skid fdp with gradle

This is 5 months old and you may not care anymore or have figured it out already, but FDP already uses gradle for compiling; gradle isn't just for Minecraft.
